assembl/static2/js/app/components/debate/survey/posts.jsx

```diff
@@ -94,9 +95,16 @@ const mapDispatchToProps = dispatch => ({
 export default compose(
   connect(mapStateToProps, mapDispatchToProps),
   graphql(QuestionPosts, {
-    options: props => ({
-      variables: { first: 8, after: '', id: props.questionId }
-    }),
+    options: (props) => {
+      const { hash } = window.location;
+      let id = null;
+      if (hash !== '') {
+        id = hash.replace('#', '').split('?')[0];
```

Comment:
Does this mean that you assume the query string will come after the hash? If so, that is against the recommended way to form a URI: https://tools.ietf.org/html/rfc3986#section-4.1

Reply:
It's potentially the share modal that add a query string at the end of the url.